bag_unpacker/src/main/scala/uk/ac/wellcome/platform/archive/bagunpacker/services/BagUnpackerWorkerService.scala

```diff
@@ -25,13 +26,21 @@ class BagUnpackerWorkerService(
     stream.run(processMessage)
 
   def processMessage(
-    unpackBagRequest: UnpackBagRequest
+    unpackBagRequest: UnpackRequest
```

Comment:
minor: rename this variable to `unpackRequest`

Reply:
I might actually put this back to `UnpackBagRequest` as it will most likely be specific to `bags` and the name will be more accurate.